libr/debug/dreg.c

```diff
@@ -83,6 +83,7 @@ R_API int r_debug_reg_sync(RDebug *dbg, int type, int write) {
 }
 
 R_API int r_debug_reg_list(RDebug *dbg, int type, int size, int rad, const char *use_color) {
+	int i;
```

Comment:
use size_t for loops, there's no need for negative indexes

Reply:
Here as well, why resolved? Also, consider moving this inside the `if`, since it is only used there.

Reply:
Ok